### DISCUSSION TITLE
want to help with testing PR's but not sure on how

### DISCUSSION THREAD

**Discussion by Rozza122** [mod-playerbots/mod-playerbots#400] [Q&A] [answered]
Hi all, i want to help testing some of the pull requests but im not sure how to get the changes from the pull requests merged into my local playerbot repo, any advice?

Thanks

**Comment by liyunfan1223** [comment, marked as answer]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

Thanks for your effort. This should help.
